app/assets/stylesheets/user-show.css

```diff
@@ -0,0 +1,91 @@
+body {
+  background-color: #814242;
```

Comment:
- **[OPTIONAL]** Kindly use another background color because of the contrast for a better UI.